website/routes.py

```diff
@@ -103,8 +106,10 @@ def get_globals():
         'reauth_url': util.web_url_for('auth_logout', redirect_url=request.url, reauth=True),
         'profile_url': cas.get_profile_url(),
         'enable_institutions': settings.ENABLE_INSTITUTIONS,
-        'keen_project_id': settings.KEEN_PROJECT_ID,
-        'keen_write_key': settings.KEEN_WRITE_KEY,
+        'keen_public_project_id': settings.KEEN['public']['project_id'],
```

Comment:
We have keys in the settings we don't want to risk leaking into mako, but point taken.  I'll update this to `keen: { public: {}, private: {}}`.